docs/tutorials/start-a-private-network-with-substrate.md

```diff
@@ -9,29 +9,33 @@ You will need:
 
 You will learn to:
 1. Compile a Substrate node from source code.
-2. Generate ed25519 key-pairs for use as a network authority
+2. Generate ed25519 and sr25519 key-pairs for use as a network authority
 3. Create and edit a chainspec json file
 
 
 Compiling the Tools
 -----------------
-Substrate is an open source framework for building your own blockchain nodes. It can be used for a very wide variety of applications.
+Substrate does not (yet) offer binary installation packages, so it must be compiled from source, which can be a time-consuming process. Commits go into the [Substrate repository](https://github.com/paritytech/substrate) regularly and it is wise that everyone participating in this network have the same version of substrate to guarantee success. In practice similar but not identical versions will often work, but banking on that is a recipe for frustration. In this tutorial, we'll be using commit `7d7e74fb` for more stability.
```

Comment:
Shall we share some context about why choose this commit, so users are more confident about their practice.

Reply:
I plan to rewrite this part in terms of Substrate Package. I'm trying to figure out the best way to get users a compatible version of subkey. This is my first checkbox. See also https://github.com/substrate-developer-hub/substrate-package/issues/25

Reply:
Phrases like "but banking on that is a recipe for frustration" probably does not translate well. See our contribution guidelines:

> Avoid using uncommon words, highly technical words, or jargon that users might not understand. Also, avoid using idioms such as "that's the way the cookie crumbles", while it might make sense to you, it may not translate well into another language. Keep in mind that a lot of users are non-native English speakers.

IMO too much detail here around versioning. I would have the text assume it does work, and a small note saying if it doesn't to open an issue.